buildSrc/src/main/kotlin/com/datadog/gradle/config/AndroidConfig.kt

```diff
@@ -20,7 +20,7 @@ object AndroidConfig {
     const val MIN_SDK_FOR_WEAR = 23
     const val BUILD_TOOLS_VERSION = "34.0.0"
 
-    val VERSION = Version(2, 13, 0, Version.Type.Snapshot)
+    val VERSION = Version(2, 13, 0, Version.Type.Release)
```

Comment:
this change should be reverted before the merge (and version should be bumped in the `develop` branch)